api/Notification.json

```
@@ -909,17 +909,23 @@
             "edge": {
               "version_added": "17"
             },
-            "firefox": {
-              "version_added": "117",
-              "notes": "Enabled by default on Windows",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.webnotifications.requireinteraction.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "117",
+                "partial_implementation": true,
+                "notes": "Only supported on Windows."
```

Should we be more explicit about what this data means?

```suggestion
                "notes": "Only supported on Windows. Behind a flag on other operating systems."
```